docs/plugins.md

```diff
@@ -1,100 +1,41 @@
 # PLUGINS
```

Comment:
Change `Plugins` to `Apps` throughout please.

Reply:
Done.